packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -0,0 +1,993 @@
+import { css, createGlobalStyle } from 'styled-components';
+
+import { rem } from '@carbon/elements';
+
+function getSelectArrowSvg(color) {
```

Comment:
@vsgoulart I decided not to remove the icon overrides for numbers and selects and move it to the core as I still think it wouldn't look right to have such Carbon related look and feel in the base styles. I think it's okay to have the overrides as is.